client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java

```diff
@@ -61,14 +61,25 @@
     private final ScheduledExecutorService scheduledExecutorService;
     private final ScheduledExecutorService cleanExpireMsgExecutors;
 
+    /**
+     * The default retry delay in milliseconds when submitting a {@link ConsumeRequest} encounters RejectedExecutionException
+     */
+    private static final int DEFAULT_CONSUME_REQUEST_LATER_DELAY_MILLS = 5000;
```

Comment:
Can it be configurable?

Reply:
Cannot be configured currently, the previous version was hard-coded with a value of 5000.
Should I make it configurable?